sdk/synapse/Azure.Analytics.Synapse.AccessControl/tests/AccessControlClientLiveTests.cs

```diff
@@ -169,37 +155,6 @@ public async Task CanListRoleDefinitions()
             Assert.GreaterOrEqual(roleDefinitionsJson.RootElement.GetArrayLength(), 1);
         }
 
-        [Test]
-        public async Task CanListRoleAssignments()
```

Comment:
It doesn't look like this test is using models - why are we deleting it?  Ditto for the others that are all only using JSON.